images/linux/scripts/installers/java-tools.sh

```diff
@@ -86,16 +78,4 @@ rm gradleLatest.zip
 ln -s /usr/share/gradle-"${gradleVersion}"/bin/gradle /usr/bin/gradle
 echo "GRADLE_HOME=/usr/share/gradle" | tee -a /etc/environment
 
-# Run tests to determine that the software installed as expected
-echo "Testing to make sure that script performed as expected, and basic scenarios work"
-for cmd in gradle java javac mvn ant; do
-    if ! command -v $cmd; then
-        echo "$cmd was not installed or found on path"
-        exit 1
-    fi
-done
-
-javaVersion=$(java -version |& head -n 1 | cut -d\" -f 2)
-javaTool "Java" $javaVersion
-javacVersion=$(javac -version |& cut -d" " -f2)
-javaTool "Javac" $javacVersion
+invoke_tests "Java" "Java"
```

Comment:
since we have the separate file for Java tests, it is enough to just write `invoke_tests "Java"` (make sure that you have pulled latest main to support this approach)

Reply:
ty, done :)